Documents/public-api.md

```diff
@@ -231,6 +231,58 @@ uli,isValid
 10Bx939c5543TqA1144M999133X38,false
 ```
 
+### Rate Spread Calculator
+
+* `rateSpread`
+
+    * `POST` - Calculate Rate Spread
+
+
+Example payload, in `JSON` format:
+
+```json
+{
+  "actionTakenType": 1,
+  "amortizationType:": 30,
```

Comment:
Minor typo that I discovered copying and pasting, there's an extra colon inside the quotes.  Reads `"amortizationType:"`, should be `"amortizationType"`